### PR TITLE
[Merged by Bors] - Airflow cron-DAG demo

### PR DESCRIPTION
**Pull request by adwk67**
## Description

A simple Airflow demo showing how to import/load a custom DAG and schedule it by enabling and triggering it.

## Review Checklist
- [x] Code contains useful comments
- [x] (Integration-)Test cases added (or not applicable)
- [x] Documentation added (or not applicable)
- [x] Changelog updated (or not applicable)
- [x] Cargo.toml only contains references to git tags (not specific commits or branches)

Once the review is done, comment `bors r+` (or `bors merge`) to merge. [Further information](https://bors.tech/documentation/getting-started/#reviewing-pull-requests)

### CODE REVIEW

File: docs/modules/ROOT/pages/demos/airflow-cron-dag.adoc [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+Superset gives the ability to execute SQL queries and build dashboards.
+Open the `airflow` endpoint `webserver-airflow` in your browser (`http://172.18.0.4:32754` in this case).
+
+image::demo-airflow-cron-dag/airflow_1.png[]
+
+Log in with the username `airflow` and password `airflow`. The overview screen shows the DAG that has been mounted during the demo set-up (`date_demo`).
+
+image::demo-airflow-cron-dag/airflow_2.png[]
+
+There are two things to notice here. The DAG has been enabled, as shown by the slider to the right of the DAG name (DAGs are initially all `paused` and can be activated either manually in the UI or via a REST call, as done in the setup for this demo):
```
Comment (sbernauer) [inline docs/modules/ROOT/pages/demos/airflow-cron-dag.adoc:53]: Not entirely sure if we're talking about the same thing....
```suggestion
There are two things to notice here. The DAG has been enabled, as shown by the slider to the left of the DAG name (DAGs are initially all `paused` and can be activated either manually in the UI or via a REST call, as done in the setup for this demo):
```

**Review by sbernauer — COMMENTED** [review]
Many thanks, demo works fine! Sorry for pushing commits, but i rebased it to main (Some internal stuff i did change on main) and fixed the nav.adoc. Website now renders fine
Only minor doc comments left. I like they way you mark your screenshots :)
Only remaining thing is maybe the name of the demo. I have the feeling that cron and dag don't make too much sense for newbies. What do you think about e.g. `airflow-scheduling`, `airflow-scheduled-job`. But i don't have a strong feeling

File: docs/modules/ROOT/pages/demos/airflow-cron-dag.adoc (hunk 1 above)
Comment (adwk67) [reply docs/modules/ROOT/pages/demos/airflow-cron-dag.adoc:53]: Good catch - this is the bane of left-handed people, BTW :)

**Review by sbernauer — APPROVED** [review]
LGTM. Feel free to think about the name described above. Don't forget to change the links to point to the main branch before merging

**Comment by adwk67** [comment]
bors merge

**Pull request closed without merging**